testsuite/dockerfiles/server-all-in-one-dev/Dockerfile

```diff
@@ -2,14 +2,14 @@ ARG BASE
 ARG VERSION
 FROM ${BASE}:${VERSION}
 
-RUN zypper ref && \
+RUN zypper -n --gpg-auto-import-keys ref && \
       zypper -n install \
       java-17-openjdk-devel \
       openssh \
       rsync \
       apache-ivy \
       ant \
-      ant-junit5 \
+      ant-junit \
```

Comment:
To be on the very safe side, we need to clarify this with Jordi, but I think this package name is also fine. I did some digging in the OBS for those 2 packages and came to the conclusion that the name without the `5` in it will work, too.